mitodl/open-discussions
# Added ES comment document indexing

#### What are the relevant tickets?
Closes #606

#### What's this PR do?
Adds logic to create/update comment documents in Elasticsearch

#### How should this be manually tested?
- Set `FEATURE_INDEX_UPDATES=True` in your `.env`
- Create/update comments and check to see if those changes are reflected in the ES index
- Set a post to removed, then check to see that all comments associated with that post are also set to removed
- Delete a comment and check to see that the parent post's `num_comments` is decremented

#### Where should the reviewer start?
Probably `search/task_helpers.py`

#### Any background context you want to provide?
There are comments that say as much, but we are going ahead with an approach where we will log information about version conflicts in document updates and let the app continue to function as normal.
